### ISSUE TITLE
How to use gluon to train model?

### ISSUE THREAD

**Issue by qingzhouzhen** [apache/incubator-mxnet#7994]
I want to use gluon to train imageclassfication network, I search for sometime, then I got the result:
1 cd to path /path/to/incubator-mxnet/example/gluon
2 then execute  `python image_classification.py --dataset=imagenet --train-data=/data/ILSVRC2012_img_train.rec --val-data=/data/ILSVRC2012_img_val.rec --gpus=4 --model=densenet169 --mode='symbolic` , It seems everything goes right.
Am I right?
Why there is not a README.md like : [https://github.com/apache/incubator-mxnet/tree/master/example/image-classification](https://github.com/apache/incubator-mxnet/tree/master/example/image-classification)
Does there need a README.md there [https://github.com/apache/incubator-mxnet/tree/master/example/gluon](https://github.com/apache/incubator-mxnet/tree/master/example/gluon) @piiswrong @szha , if does, I can write one.

**Comment by szha** [comment]
Thanks. It's a good idea to have a readme file. Thank you for offering to help.

**Comment by zhreshold** [comment]
Be aware that lr_scheduler is not included, so the learning-rate is constant.
And the model is only saved at the end of the training, which may not be appropriate for long time imagenet training.
@qingzhouzhen  You are very welcome to contribute

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by szha** [comment]
Hi @qingzhouzhen. Let us know if you need more input from us.
